autoload/vebugger/std.vim

```diff
@@ -301,17 +301,19 @@ function! s:standardCloseHandlers.removeCurrentMarker(debugger) dict
     sign unplace 1
 endfunction
 
+let s:breakpoint_text = get(g:, 'vebugger_breakpoint_text', '->')
+let s:currentline_text = get(g:, 'vebugger_currentline_text', '**')
 
 if hlexists("DebuggedLine")
   sign define vebugger_current linehl=DebuggedLine
 else
-  sign define vebugger_current text=->
+  exe 'sign define vebugger_current text=' . s:currentline_text
```

Comment:
Please use full command names in plugins...

Reply:
ok, done